docs/profiles/profile-single-tenant-cluster.md

```diff
@@ -0,0 +1,93 @@
+# Kubernetes Multi-tenancy profile: Single Tenant Cluster
+
+In order to define a secure multi-tenant cluster, we must first agree upon a baseline for a secure single tenant cluster. 
+
+## Overall functional model summary
+
+1. A **Tenant** is defined as a team of users/ identities that shall have exclusive use of one or more resources of a Kubernetes cluster in parallel with users of other tenants using their own separate and isolated resources within the same cluster. We will interchangeably use the terms \"users\" or \"members\" of a tenant.
+
+2. In this single tenant profile, a Kubernetes cluster and all of its objects and resources are explicitly owned only by a single tenant. There is no sharing of the cluster or attempt to wall off objects and resources between different tenant users.
+
+## Applicable benchmarks used to define a secure Kubernetes cluster
+
+1. NIST SP 800-190: Application Container Security Guide https://csrc.nist.gov/publications/detail/sp/800-190/final 
+1. CIS Benchmark for Kubernetes 1.4 https://csrc.nist.gov/publications/detail/sp/800-190/final
+
+## Kubernetes cluster configuration requirements for profile single tenant
+
+1. The recommended minimum version of Kubernetes release is 1.15. Earlier kubernetes releases may be used with some potential restrictions.  (Note: Maybe better to list api versions required for various Kubernetes resources/ apis).
```

Comment:
A lot of the features required below are still in beta. May want to call that out specifically that we are recommending non-GA APIs and why it's ok.